test/ninja/ninja_test_sconscripts/sconstruct_no_for_sig_subst

```diff
@@ -0,0 +1,24 @@
+import SCons
+
+SetOption('experimental','ninja')
+DefaultEnvironment(tools=[])
+
+env = Environment()
```

Comment:
will `tools=[]` allow this test to work? if so,please add.

Reply:
fixed: 
3022d5b